makefile

```diff
@@ -109,6 +114,12 @@ wp8 ?= $(root)/wp8
 
 classpath = avian
 
+bootimage-classpath = $(classpath-build)
+
+ifeq ($(bootimage-test),true)
```

Comment:
Yeah, I wasn't sure exactly what kind of command line interface I wanted
for this, and I just went for what took the least amount of effort.  The
good news is that if you specify bootimage-test=true without
bootimage=true, nothing unusual will happen, since bootimage-classpath will
simply go unused.

I guess we could either set bootimage=true automatically if
bootimage-test=true is specified, or else complain that the latter was
specified without the former.  Which would you prefer?

On Mon, Oct 19, 2015 at 3:25 PM, Joshua Warner <notifications@github.com>
wrote:

> In makefile
> <https://github.com/ReadyTalk/avian/pull/459#discussion_r42428000>:
>
> > @@ -109,6 +114,12 @@ wp8 ?= $(root)/wp8
> >
> >  classpath = avian
> >
> > +bootimage-classpath = $(classpath-build)
> > +
> > +ifeq ($(bootimage-test),true)
>
> To be consistent with the adjustment of options above, we might want to
> wrap this in a ifeq ($(bootimage),true). Otherwise, you could
> accidentally build with bootimage-test, but not bootimage and get
> unexpected results.
>
> Better yet, maybe add a check to the parameter validation code.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ReadyTalk/avian/pull/459/files#r42428000>.
>